recipes/python-gmail/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "python-gmail" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3a8346398b42daf0f2f123a0e10eb31a7f23cd85b5ac7429fdb31d9fb209a1c0
+
+build:
+  skip: False  # [win]
```

Comment:
You don't want to skip windows. You should just make it `noarch: python` since there is no specific dependencies or build procedure for windows. The test will still fail, but that's ok for the reasons I've mentioned elsewhere on this PR.